codestodians/komlie-frontend
# Feature(166545228): Display products

#### What does this PR do?
- Create Products Page

#### Description of Task to be completed?
Products should be displayed when a Category is clicked

#### How should this be manually tested?
 ON the api repo:
- Pull latest changes and install all upgraded packages.
- Ensure Database name is `komlie`
- Reach out for the content of db.json
- run ./manage.py loaddata db.json
- run the server

ON the frontend:
- run npm install
- npm start
- Click on `Cosmetics` category

#### Any background context you want to provide?
- Ensure you reach out for the content of db.json
- Note that the data do not persist yet so when the app is reloaded, the products will vanish. Another story will be created to fix that. 


#### What are the relevant pivotal tracker stories?
[#166545228](https://www.pivotaltracker.com/story/show/166545228)


#### Screenshots (if appropriate)

#### Questions:
